app/src/main/res/layout/save_tab_group_button.xml

```diff
@@ -2,24 +2,11 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<com.google.android.material.button.MaterialButton xmlns:android="http://schemas.android.com/apk/res/android"
+<com.google.android.material.button.MaterialButton
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    style="@style/PositiveButton"
     android:id="@+id/save_tab_group_button"
-    style="@style/Widget.MaterialComponents.Button.TextButton"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:minHeight="36dp"
-    android:insetTop="0dp"
-    android:insetBottom="0dp"
     android:layout_marginBottom="20dp"
-    android:background="@drawable/button_background"
-    android:letterSpacing="0"
     android:text="@string/save_to_collection"
-    android:textAllCaps="false"
-    android:textColor="?contrastText"
-    app:backgroundTint="?accent"
-    app:icon="@drawable/ic_tab_collection"
-    app:iconGravity="textStart"
-    app:iconPadding="8dp"
-    app:fontFamily="@font/metropolis_medium"
```

Comment:
You may want to verify with UX cases like this here in that this button was one of the few and only cases in which semi-bold was too bold and they requested medium instead. We have iterated on this a couple of times though, but may want to check with them to see whether they're happy with it in semi-bold or medium. In your implementation, you inherit the styling for the positive button from the neutral one, there may be a couple more cases like this as you go through headers etc. later where it may require a different font than from the default button.

Reply:
I will check with them before I get UX sign off! 😄